### PR TITLE
US14_SayoraNegmatova added new code

### PR DESCRIPTION
**Pull request by Sayora1**

### CODE REVIEW

**Review by SForogh — APPROVED** [review]
Looks perfect! Great job Sayora.

**Comment by hamidalakbarov** [comment]
Great Job Sayora!

**Review by LinaVisochin — APPROVED** [review]

File: src/test/resources/features/US14_PhotosModule.feature [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  Scenario: Verify user can access to the Photos module
+    When user clicks the "Photos" module
+    Then user can see following modules
+      | Your photos     |
+      | Your videos     |
+      | Favorites       |
+      | On this day     |
+      | Your folders    |
+      | Shared with you |
+      | Tagged photos   |
```
Comment (vsimpryvit) [inline src/test/resources/features/US14_PhotosModule.feature:16]: Good Job Sayora! Thanks for review request:)

**Review by vsimpryvit — APPROVED** [review]
Great job Sayora! Good to go!

**Comment by sajawii** [comment]
Look good

**Comment by hamidalakbarov** [comment]
Great work Sayora!

**Pull request merged**